src/sentry/integrations/on_call/metrics.py

```diff
@@ -0,0 +1,61 @@
+from collections.abc import Mapping
```

Comment:
nit: is there a reason why this file is in it's own dir ? we could move it up one and call it on_call_metrics

Reply:
Followed the convention set by messaging integrations, but happy to move it since there are only two on call integrations.

Reply:
gotchas, do we have more on_call modules planned in the future? Then it would make sense to keep this